File: lua/refactoring/tests/refactor/106/lua/var-no-assignment/extract.expected.lua
-- stylua: ignore start

local function foo_bar(a, test, test_other)
    for idx = 0, test_other do
        print(idx, a, test)
    end
end


function simple_function(a)
    local test
    local test_other = 11
    foo_bar(a, test, test_other)
end

File: lua/refactoring/tests/refactor/106/rb/simple-function/extract.expected.rb

def foo_bar(a, test, test_other)
  [test, test_other].each do |v|
    puts "#{a} #{v}"
  end
end

def simple_function(a)
  test = 1
  test_other = 11

  foo_bar(a, test, test_other)
end
